### PR TITLE
ADR: Introduce working groups

### PR DESCRIPTION
**Pull request by cescoffier**
- Define the concept of working groups to organize work around specific topics.
- Establish simple criteria for defining working groups.
- Outline the process for initiating, organizing, and driving working groups.
- Detail the roles and communication methods for working group points of contact.
- Include guidelines for participation, communication, and updating working groups.
- Specify the lifecycle of a working group from definition to completion.
- Address the advantages and potential drawbacks of implementing working groups.
- Compare working groups to the current workflow and other considered options.

### CODE REVIEW

File: adr/0005-focus-group-process.adoc [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+We also need to ensure that the work is done transparently and that the community can participate in the discussions.
+
+This would be the basis of an informal roadmap, where the community can see what is being worked on and what is coming next.
+Our previous attempts to publish and maintain a roadmap were not successful.
+We need a more lightweight approach, focusing on current work and next steps.
+
+Additionally, new contributors may find it hard to find a way to contribute to the project, as the project's size may be overwhelming.
+Working on a specific area or topic may be more appealing to new contributors and may help them get started.
+
+== Focus Groups
```
Comment (maxandersen) [inline adr/0005-focus-group-process.adoc:21]: i've looked in dictionaries and definitions and I think we should reconsider using focus group.

alternatives:

- task force
- action group
- working group 

?

File: adr/0005-focus-group-process.adoc [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+=== Point of Contact and Communication
+
+The point of contact is the main entry point for the focus group.
+Both the community and the team can reach out to this person to get more information about the focus group or to participate.
+The point of contact should be available on GitHub and Zulip, ensuring that communication is done transparently.
+A focus group may have multiple points of contact, depending on the size and scope of the group.
+
+Most communication should be done on GitHub discussions, issues, and PRs.
+If the focus group needs to organize calls, these calls should be open to everyone in the community.
+It is important for the focus group to publish the outcome of these discussions and decisions made during these calls.
```
Comment (maxandersen) [inline adr/0005-focus-group-process.adoc:68]:
```suggestion
It is important for the focus group to publish the outcome of these discussions and possible decisions made during these calls. 
```

File: adr/0005-focus-group-process.adoc (hunk 1 above)
Comment (cescoffier) [reply adr/0005-focus-group-process.adoc:21]: Well, none of these looks good to me.

Task force - that's not the right meaning. Force is not what we want to convey.

Working group - it's portably the less bad, but comes with a connoted process in mind. 

Action group - not sure what it means, but 'action' like 'force' is a bit too close to military wording.

At some point we mentioned focused initiatives. A bit long, but convey the right meaning.

Comment (cescoffier) [reply adr/0005-focus-group-process.adoc:21]: Working group has an advantage over the others. Using the short form (wg-) is relatively well understood. 

For example: wg-websocket-next would be relatively well understood.

Comment (maxandersen) [reply adr/0005-focus-group-process.adoc:21]: true, and fi-websocket-next for "focused initiative" sounds like financial bitcoin mining.

I'm fine with working group

File: adr/0005-focus-group-process.adoc [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+The idea behind this proposal is to introduce the concept of _focus groups_.
+A focus group is a lightweight way to organize work around a specific topic.
+It aims to gather people interested in a specific topic and ensure that the work is done transparently.
+It also aims to ensure that the community is aware of ongoing work and can participate in the discussions.
+
+=== Defining a Focus Group
+
+To initiate a focus group, we need to define the scope of the focus group.
+This should be a specific topic or initiative that needs to be worked on.
+The definition of a focus group should follow the _SMART_ criteria:
```
Comment (maxandersen) [inline adr/0005-focus-group-process.adoc:32]: been thinking on this mention of SMART (which btw. for me is not exactly equal to "light weight":) 

are you suggesting these gets called out in the discussion or just a good goal? 

I can do most of them fairly easily except measurable...are you just asking for it being trackable ? as measurable from SMART is normally "15% faster than X" or some other quantity not just "when all issues are resolved".

File: adr/0005-focus-group-process.adoc (hunk 1 above)
Comment (cescoffier) [reply adr/0005-focus-group-process.adoc:21]: Yes, let me change to working group.

File: adr/0005-focus-group-process.adoc (hunk 3 above)
Comment (cescoffier) [reply adr/0005-focus-group-process.adoc:32]: I don't like mentioning SMART either. I just needed a way to explain how to scope a working group, to avoid the: let's rebuild the universe. 

Let me rephrase it a bit.

Comment (cescoffier) [reply adr/0005-focus-group-process.adoc]: Done. LTM.

File: adr/0005-focus-group-process.adoc (hunk 1 above)
Comment (cescoffier) [reply adr/0005-focus-group-process.adoc]: Done.

**Review by gastaldi — APPROVED** [review]

**Comment by cescoffier** [comment]
@maxandersen Ping.

**Review by maxandersen — COMMENTED** [review]
Looks good. Should provide links to the overview page when we have it.

**Review by maxandersen — APPROVED** [review]
